scheduler/server/orig_scheduling_alg.go

```diff
@@ -7,10 +7,42 @@ import (
 	"github.com/twitter/scoot/scheduler/domain"
 )
 
-type OrigSchedulingAlg struct{}
```

Comment:
I'm surprised this files contents changed. Shouldn't it stay the same mostly if the impl hasn't changed?